sdk/src/loom2d/Display/Graphics.ls

```diff
@@ -74,36 +101,128 @@ package loom2d.display
     }
     
     [Native(managed)]
+    /**
+     * Draw GPU accelerated vector graphics. Used in conjunction with the Shape class.
+     *
+     * Draw calls are buffered and drawn every frame until you call clear().
+     */
     public native class Graphics
     {
         public function Graphics()
-        {            
-            
+        {
         }
         
+        /**
+         * Clear all draw commands and return to a clean state.
+         */
         public native function clear():void;
+        
+        /*
+         * Specify the style used for lines.
+         */
         public native function lineStyle(thickness:Number = NaN, color:uint = 0x00000000, alpha:Number = 1, pixelHinting:Boolean = false, scaleMode:String = "", caps:String = "round", joints:String = "round", miterLimit:Number = 0):void;
+        
+        /**
+         * Indicate the format used to display text.
+         */
         public native function textFormat(format:TextFormat):void;
+        
+        /**
+         * Get the bounds of a given string as it would be drawn with specified format.
+         */
         public native function textLineBounds(format:TextFormat, x:Number, y:Number, text:String):Rectangle;
+        
+        /**
+         * Determine how far a given string will move to the right.
```

Comment:
Determine the advance (logical width) of the given string. This can vary from the bounds and is usually used for while positioning text.